daemons/compute/client-go/main.go

```diff
@@ -105,11 +105,15 @@ func main() {
 		// Poll request to check for completed/cancelled
 		for {
 			statusResponse, err := getStatus(ctx, c, *workflow, *namespace, uid, *maxWaitTime)
+			if err != nil {
+				log.Fatalf("failed to get dat movement status: %v", err)
```

Comment:
```suggestion
				log.Fatalf("failed to get data movement status: %v", err)
```

Reply:
Also, some log messages are capitalized and some are not in the function. Might be my own doing.